Dockerfile

```diff
@@ -4,8 +4,9 @@ USER root:root
 
 RUN apk add --no-cache curl && \
     curl -L https://storage.googleapis.com/kubernetes-release/release/v1.7.5/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl && \
-    chmod +x /usr/local/bin/kubectl && kubectl version --client
-
+    chmod +x /usr/local/bin/kubectl && kubectl version --client && \
+    curl -L /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 && \
+    chmod +x /usr/local/bin/dumb-init
 COPY kube-scheduler-healthcheck /usr/local/bin/
-
+ENTRYPOINT ["/usr/bin/dumb-init", "--", "./exechealtz"]
```

Comment:
Upstream entrypoint according to `docker inspect gcr.io/google-containers/exechealthz-amd64:v1.2.0` is `"/exechealthz"`

Reply:
thanks, was trying to wade through the Makefile to figure it out, didn't know about docker inspect